src/javaharness/java/arcs/webimpl/WebHarnessController.java

```diff
@@ -0,0 +1,83 @@
+package arcs.webimpl;
+
+import arcs.api.ArcsEnvironment;
+import arcs.api.DeviceClient;
+import elemental2.dom.*;
```

Comment:
What is this, and is it a new dependency?

Reply:
Elemental is the glue between Java and the WebAPIs for GWT and J2CL. Think of it as the Java equivalent of Typescript's .d.ts files. It has zero code-size and run-time overhead.